metal/config.go

```diff
@@ -59,5 +64,13 @@ func (c *Config) Client() *packngo.Client {
 	retryClient.RetryWaitMax = c.MaxRetryWait
 	retryClient.CheckRetry = packngo.RetryPolicy
 	httpClient := retryClient.StandardClient()
-	return packngo.NewClientWithAuth(consumerToken, c.AuthToken, httpClient)
+
+	client := packngo.NewClientWithAuth(consumerToken, c.AuthToken, httpClient)
+	tfUserAgent := httpclient.TerraformUserAgent(c.terraformVersion)
```

Comment:
https://www.terraform.io/docs/extend/guides/v1-upgrade-guide.html#deprecations suggested this function exists for the v2 SDK, it appears to be in the (unversioned) v1 SDK, identical to what we had. Looks like we don't have to duplicate it.

See 9692ab0

Reply:
:+1: